### ISSUE TITLE
Library is abandoned and broken

### ISSUE THREAD

**Issue by dtilchin** [rafael-munoz/rollbar-java#8]
Please close this issue if anyone (the author perhaps?) is interested in fixing this project.

There several bugs with no response in over a year, some of them totally breaking the library:
https://github.com/rafael-munoz/rollbar-java/issues/6
https://github.com/rafael-munoz/rollbar-java/issues/2 (there's a PR open for this from months ago with no response)

As far as I can tell, this library is abandoned and totally nonfunctional. I am largely creating this issue to serve as a warning to others, so you can avoid wasting a bunch of time like I did.

**Comment by contentfree** [comment]
Since Rollbar has wire-compatibility with Airbrake API (v2.2, I think), we might be able to use the https://github.com/airbrake/airbrake-java library instead of this one.

**Comment by dtilchin** [comment]
There are quite a few forks and forks-of-forks-of-forks of this library as well. I have been using https://github.com/tapstream/rollbar-logback with no issues so far. https://github.com/fstech/rollbar-logback appears to be a more recent fork of that, with some improvements.

This library is dead; Rollbar has already removed the link to it from their site.

**Comment by contentfree** [comment]
Weird. Rollbar's introductory "Set up a notifier" after signing up has no mention of anything besides this library. Going to Rollbar's doc site and then clicking Java has more options. I'll email them.

**Comment by dtilchin** [comment]
I emailed them saying this library is nonfunctional the other day and they removed it at least from the "Other Integrations" page. There is the link to rollbar-logback at least. And, there is a possibility that there will be an official library at some point, which would be great.

**Comment by contentfree** [comment]
Yeah, I just got an email back from them saying that they're updating the Getting Started page which directed me here. Unfortunately weren't not using logback (still on log4j) so I can't use your suggestion. Looks like it's time to run through the forks! Thanks for the heads up
